app/[locale]/projects/page.tsx

```diff
@@ -42,12 +42,23 @@ const ProjectsPage = () => {
     setLoading(true);
     const existingTags = structuredClone(tags);
 
-    // we set tag active to true
-    for (const exisitingTag of existingTags) {
+    // we set tags chosen to active and all the others to not
+    if (existingTags.every(value => value.isActive)) {
+      existingTags.forEach(existingTag => {
```

Comment:
Instead of putting comment explaining your code you could extract this part into a function and call it